web/packages/core/src/internal/player/inner.tsx

```diff
@@ -858,6 +848,32 @@ export class InnerPlayer {
         }
     }
 
+    // The effective config loaded upon `load`, possibly changing if `window.RufflePlayer` changes later.
+    get loadedConfig(): URLLoadOptions | DataLoadOptions | undefined {
+        if (this.options) {
+            return {
+                ...DEFAULT_CONFIG,
+                // The default allowScriptAccess value for polyfilled elements is samedomain.
+                ...(this.isPolyfillElement && "url" in this.options
+                    ? {
+                          allowScriptAccess: parseAllowScriptAccess(
+                              "samedomain",
+                              this.options.url,
+                          )!,
+                      }
+                    : {}),
+                ...(window.RufflePlayer?.config ?? {}),
+                // This will make extension config take precedence, if I move this one line higher self-hosted config will take precedence
+                // Let the discussions begin
```

Comment:
Comment is here to facilitate discussion. Start discussing below. This has come up before. Site owners have come out against it, mainly arguing against `showSwfDownload` being able to be changed by end-users. However, it is true that end users can usually just check the network tab, making this just a convenience method.

Reply:
The extension config taking precedence is fine with me 👍